rsciio/hspy/specifications.yaml

```diff
@@ -1,4 +1,5 @@
 format_name: HSPY
+format_alias: HSpy
```

Comment:
Would it be better to support case insensitive `format_name` and avoid manually adding an alias with upper/lower case?

Reply:
Yes, in fact HyperSpy does not care about the case as it always handles the field with a `.lower()` -- so in fact we could just remove the Alias from the three formats affected.

There was just the inconsistency between `HSPY` and `ZSpy` as format names.

Reply:
I would leave it empty then.